db/schema.rb

```diff
@@ -15,7 +15,7 @@
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
-  create_table "active_storage_attachments", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+  create_table "active_storage_attachments", id: :uuid, default: -> { "public.gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
Why have all of these changed?

Reply:
D'oh, I ran migration when I had a production restored DB, not a built from scratch version... I'll get that rebased and revoked :+1: